crates/testing/src/helpers.rs

```diff
@@ -145,10 +137,10 @@ pub async fn build_cert<
     V: Versions,
     DATAType: Committable + Clone + Eq + Hash + Serialize + Debug + 'static,
     VOTE: Vote<TYPES, Commitment = DATAType>,
-    CERT: Certificate<TYPES, Voteable = VOTE::Commitment>,
+    CERT: Certificate<TYPES, VOTE::Commitment, Voteable = VOTE::Commitment>,
 >(
     data: DATAType,
-    membership: &TYPES::Membership,
+    da_membership: &TYPES::Membership,
```

Comment:
ignore the previous remark, but I think this should stay

```suggestion
    membership: &TYPES::Membership,
```

Reply:
7aaf224f1fb6bd306101cd2900e0534224255ce6